### ISSUE TITLE
“View Data” doesn't see a fresh log file unless app force closed and reopened

### ISSUE THREAD

**Issue by astralbat** [adamlechowicz/Backtrack#2]
Hello,

Great simple app!

I find that after tracking for a couple of days I have never been able to view data. I am using iCloud drive and I can see the file being written there, seemingly without any problems.

**Comment by astralbat** [comment]
Ah, I have found the workaround/solution.

After force-closing the application, I am now able to view data. So I suppose the file is only checked for on initialisation.

**Comment by adamlechowicz** [comment]
Yeah, this is how it's implemented.  There's probably a better way to do this -- naively you'd constantly check for the file, but that has negative effect on battery life and performance.  I will probably get around to fixing this issue and the other known issue sometime this summer, when I have some free time.  Glad you're enjoying the app!
